src/main/java/screens/Window.java

```diff
@@ -0,0 +1,156 @@
+package screens;
```

Comment:
I think it is kind of strange that you created a new class that extends Application while we already had App class for that. You could have changed that one.